lib/optimize/InnerGraphPlugin.js

```diff
@@ -146,6 +149,14 @@ class InnerGraphPlugin {
 								const name = statement.id ? statement.id.name : "*default*";
 								const fn = InnerGraph.tagTopLevelSymbol(parser, name);
 								statementWithTopLevelSymbol.set(statement, fn);
```

Comment:
If a `class` has non-pure super class expression, the super class must be walked without top level symbol set (`InnerGraph.setTopLevelSymbol`).

This seem to be impossible to handle with the current system and would require some refactoring